docs/monitoring.md

```diff
@@ -821,7 +821,6 @@ A list of the available metrics, with a short description:
 Executor-level metrics are sent from each executor to the driver as part of the Heartbeat to describe the performance metrics of Executor itself like JVM heap memory, GC information.
 Executor metric values and their measured memory peak values per executor are exposed via the REST API in JSON format and in Prometheus format.
 The JSON end point is exposed at: `/applications/[app-id]/executors`, and the Prometheus endpoint at: `/metrics/executors/prometheus`.
-The Prometheus endpoint is conditional to a configuration parameter: `spark.ui.prometheus.enabled=true` (the default is `false`).
```

Comment:
Hmm, why we need to remove it from document?

Reply:
The previous sentence already mentioned it. And, this sentence is invalid because we enabled it by default, @viirya .

Reply:
Oh, I means that the config now is not mentioned in this doc. Is any other place we have mentioned this config?

Reply:
Ah, for that, I can add it to the config table officially.

Reply:
Actually, a few other UI configs are missing too. I can handle them together.

Reply:
Ok, thanks.